transformers/integration_tests/cat_file_test.go

```diff
@@ -203,6 +204,46 @@ var _ = Describe("Cat File transformer", func() {
 			ContractAddresses: []string{test_data.Cat110Address()},
 			ContractAbi:       constants.Cat110ABI(),
 		}
+		It("persists a chop dunk event (dunk)", func() {
+			chopDunkBlockNumber := int64(10769102)
+			header, err := persistHeader(db, chopDunkBlockNumber, blockChain)
+			Expect(err).NotTo(HaveOccurred())
+			catFileConfig.TransformerName = constants.CatFileChopDunkTable
+			catFileConfig.Topic = constants.CatFileChopDunkSignature()
+			catFileConfig.StartingBlockNumber = chopDunkBlockNumber
+			catFileConfig.EndingBlockNumber = chopDunkBlockNumber
+
+			initializer := event.ConfiguredTransformer{
+				Config:      catFileConfig,
+				Transformer: chop_dunk.Transformer{},
+			}
+			transformer := initializer.NewTransformer(db)
+
+			logs, err := logFetcher.FetchLogs(
+				[]common.Address{common.HexToAddress(catFileConfig.ContractAddresses[0])},
+				[]common.Hash{common.HexToHash(catFileConfig.Topic)},
+				header)
+			Expect(err).NotTo(HaveOccurred())
+
+			eventLogs := test_data.CreateLogs(header.Id, logs, db)
+
+			err = transformer.Execute(eventLogs)
+			Expect(err).NotTo(HaveOccurred())
+
+			var dbResult catFileChopDunkModel
+			msgSenderID, err := shared.GetOrCreateAddress("0xBE8E3e3618f7474F8cB1d074A26afFef007E98FB", db)
+			Expect(err).NotTo(HaveOccurred())
+			err = db.Get(&dbResult, `SELECT address_id, msg_sender, what, data FROM maker.cat_file_chop_dunk`)
+			Expect(err).NotTo(HaveOccurred())
+			addressID, err := shared.GetOrCreateAddress(test_data.Cat110Address(), db)
+			Expect(err).NotTo(HaveOccurred())
+
+			Expect(dbResult.AddressID).To(Equal(addressID))
+			Expect(dbResult.MsgSenderID).To(Equal(msgSenderID))
+			Expect(dbResult.What).Should(Or(Equal("dunk"), Equal("chop")))
```

Comment:
This is the only block with `dunk` events, but it also has `chop` events (same signature). Not sure if there's a better assertion here.

Reply:
I would use a `db.Select` to get both results and assert that `len(dbResults` == 2 - so that you know we're not just matching the chop event